data/json/recipes/recipe_weapon.json

```diff
@@ -316,16 +316,81 @@
     "category": "CC_WEAPON",
     "subcategory": "CSC_WEAPON_PIERCING",
     "skill_used": "fabrication",
-    "time": "1 m",
+    "time": "2 m",
     "reversible": true,
     "autolearn": true,
     "components": [
-      [ [ "stick_long", 1 ], [ "broom", 1 ], [ "pool_cue", 1 ] ],
-      [ [ "spike", 1 ] ],
+      [ [ "stick_long", 1 ], [ "long_pole", 1 ], [ "broom", 1 ], [ "pool_cue", 1 ] ],
+      [
+        [ "spike", 1 ],
+        [ "knife_butcher", 1 ],
+        [ "knife_chef", 1 ],
+        [ "knife_carving", 1 ],
+        [ "knife_vegetable_cleaver", 1 ],
+        [ "knife_meat_cleaver", 1 ],
+        [ "blade", 1 ],
+        [ "knife_combat", 1 ],
+        [ "knife_hunting", 1 ],
+        [ "knife_rambo", 1 ],
+        [ "knife_rm42", 1 ],
+        [ "knife_trench", 1 ],
+        [ "tanto", 1 ],
+        [ "kirpan", 1 ]
+      ],
       [ [ "rag", 1 ], [ "felt_patch", 1 ], [ "leather", 1 ], [ "fur", 1 ] ],
       [ [ "cordage", 1, "LIST" ], [ "duct_tape", 50 ] ]
     ]
   },
+  {
+    "type": "recipe",
+    "result": "spear_knife_superior",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_PIERCING",
+    "skill_used": "fabrication",
+    "difficulty": 1,
+    "time": "30 m",
+    "reversible": true,
+    "autolearn": true,
+    "qualities": [ { "id": "CUT", "level": 1 }, { "id": "HAMMER", "level": 1 }, { "id": "DRILL", "level": 1 } ],
+    "components": [
+      [ [ "stick_long", 1 ], [ "long_pole", 1 ], [ "pool_cue", 1 ] ],
+      [
+        [ "knife_butcher", 1 ],
+        [ "knife_chef", 1 ],
+        [ "knife_carving", 1 ],
+        [ "knife_vegetable_cleaver", 1 ],
+        [ "knife_meat_cleaver", 1 ],
+        [ "blade", 1 ],
+        [ "knife_combat", 1 ],
+        [ "knife_hunting", 1 ],
+        [ "knife_rambo", 1 ],
+        [ "knife_rm42", 1 ],
+        [ "knife_trench", 1 ],
+        [ "tanto", 1 ],
+        [ "kirpan", 1 ]
+      ],
+      [ [ "nail", 2 ] ],
+      [ [ "cordage", 2, "LIST" ], [ "duct_tape", 100 ] ]
+    ]
+  },
+  {
```

Comment:
The primitive recipie allows a crude spike to be used while the superior one don't. Upgrading simple to superior bypasses this.

Reply:
I didn't really want to remove the ability to make an emergency spear with a spike, but I don't think it's appropriate for a superior spear. Found a workaround by making an even crappier knife spear option.